lib/PuppeteerSharp/Page.cs

```diff
@@ -63,28 +76,97 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
             Client.MessageReceived += client_MessageReceived;
         }
 
+        internal Session Client { get; }
+
         #region Public Properties
+
+        /// <summary>
+        /// Emitted when the JavaScript <c>load</c> <see cref="https://developer.mozilla.org/en-US/docs/Web/Events/load"/> event is dispatched.
+        /// </summary>
         public event EventHandler<EventArgs> Load;
+
+        /// <summary>
+        /// Emitted when the page crashes
+        /// </summary>
         public event EventHandler<ErrorEventArgs> Error;
+
+        /// <summary>
+        /// Emitted when the JavaScript code makes a call to <c>console.timeStamp</c>. For the list of metrics see <see cref="Page.MetricsAsync"/>.
+        /// </summary>
         public event EventHandler<MetricEventArgs> MetricsReceived;
+
+        /// <summary>
+        /// Emitted when a JavaScript dialog appears, such as <c>alert</c>, <c>prompt</c>, <c>confirm</c> or <c>beforeunload</c>. Puppeteer can respond to the dialog via <see cref="PuppeteerSharp.Dialog"/>'s <see cref="Dialog.Accept(string)"/> or <see cref="Dialog.Dismiss"/> methods.
+        /// </summary>
         public event EventHandler<DialogEventArgs> Dialog;
+
+        /// <summary>
+        /// Emitted when JavaScript within the page calls one of console API methods, e.g. <c>console.log</c> or <c>console.dir</c>. Also emitted if the page throws an error or a warning.
+        /// The arguments passed into <c>console.log</c> appear as arguments on the event handler.
+        /// </summary>
+        /// <example>
+        /// An example of handling <see cref="Console"/> event:
+        /// <code>
+        /// page.Console += (sender, e) => 
+        /// {
+        ///     for (var i = 0; i < e.Message.Args.Count; ++i)
+        ///     {
+        ///         System.Console.WriteLine($"{i}: {e.Message.Args[i]}");
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
         public event EventHandler<ConsoleEventArgs> Console;
 
+        /// <summary>
+        /// Emitted when a frame is attached.
+        /// </summary>
         public event EventHandler<FrameEventArgs> FrameAttached;
+
+        /// <summary>
+        /// Emitted when a frame is detached.
+        /// </summary>
         public event EventHandler<FrameEventArgs> FrameDetached;
+
+        /// <summary>
+        /// Emitted when a frame is navigated to a new url.
+        /// </summary>
         public event EventHandler<FrameEventArgs> FrameNavigated;
 
+        /// <summary>
+        /// Emitted when a <see cref="Response"/> is received.
+        /// </summary>
         public event EventHandler<ResponseCreatedEventArgs> ResponseCreated;
```

Comment:
I think that this should be rename to `Response` to match puppeteer

Reply:
Same